### PR TITLE
add screenshots to images folder for proper wiki page display

### PR DESCRIPTION
**Pull request by jx2014**
add folders "images".
add screenshots under "images" folder so that the wiki page can display those images.
move "example.svg" under "images" folder.

### CODE REVIEW

File: images/005.png
Comment (Finomnis) [inline images/005.png:1]: Probably should not enable "print on both sides"

**Comment by Finomnis** [comment]
Looks good to me. Not sure if you want to keep the "print on both sides" in the screenshots, but if you say it does not matter, we can merge

File: images/005.png
Comment (jx2014) [reply images/005.png:1]: Good catch. I am creating a new screenshot with it unchecked.

**Pull request merged**